src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -42,8 +42,6 @@ export default Object.freeze({
   searchTypeaheadEnabled: 'search_typeahead_enabled',
   sharableLinks: 'sharable_link',
   show526Wizard: 'show526Wizard',
-  showChapter31: 'show_chapter_31',
-  showChapter36: 'show_chapter_36',
```

Comment:
Unrelated, but the chapter 36 feature flag was removed some time ago and is no longer in use.